### PR TITLE
Fix compilation issue of lib-subgraph

### PR DESCRIPTION
**Pull request by wollooo**
https://github.com/kumodao/kumo-protocol/issues/176

### CODE REVIEW

File: packages/contracts/contracts/Interfaces/IKUMOStaking.sol [hunk 1]
```diff
@@ -35,8 +35,7 @@ interface IKUMOStaking {
         address _kusdTokenAddress,
         address _troveManagerAddress, 
         address _borrowerOperationsAddress,
-        address _activePoolAddress,
-        address _treasuy
+        address _activePoolAddress
```
Comment (udosson) [inline packages/contracts/contracts/Interfaces/IKUMOStaking.sol:38]: Should this change be included in this PR?

File: packages/contracts/contracts/KUMO/KUMOStaking.sol [hunk 2]
```diff
@@ -77,15 +77,14 @@ contract KUMOStaking is IKUMOStaking, Ownable, CheckContract, BaseMath {
         address _kusdTokenAddress,
         address _troveManagerAddress, 
         address _borrowerOperationsAddress,
-        address _activePoolAddress,
-        address _treasury
+        address _activePoolAddress
```
Comment (udosson) [inline packages/contracts/contracts/KUMO/KUMOStaking.sol:80]: Should this change be included in this PR?

**Review by udosson — COMMENTED** [review]
@wollooo - is this commit needed to fix issue #177?
https://github.com/kumodao/kumo-protocol/pull/177/commits/45b5d99e5cf218a2df1c9b3ab5720a3b2bc1de5e

If not, please remove it from this PR, and let's have a clean git history. thx

File: packages/contracts/contracts/Interfaces/IKUMOStaking.sol (hunk 1 above)
Comment (wollooo) [reply packages/contracts/contracts/Interfaces/IKUMOStaking.sol]: no, this only fixes some errors when running the tests

File: packages/contracts/contracts/KUMO/KUMOStaking.sol (hunk 2 above)
Comment (wollooo) [reply packages/contracts/contracts/KUMO/KUMOStaking.sol]: no, this only fixes some errors when running the tests

**Comment by wollooo** [comment]
the correct commit is: https://github.com/kumodao/kumo-protocol/pull/177/commits/da164f8d89d7b88a653866bb2cd3ce18c5534e50

**Comment by udosson** [comment]
@wollooo could you remove the commits related to `KumoStaking`, please?
You could use `git reset --soft HEAD~2` and then force push.
We can also do this together in our devs call today

**Pull request merged**